### ISSUE TITLE
Suggestion about italics

### ISSUE THREAD

**Issue by gutofarias** [rougier/nano-emacs#59]
I noticed that the italics style and color is defined by nano-face-faded. But in your article you said that faded "is for secondary information that is less important". Which I think is the opposite case of italics, we should use it to emphasize important stuff. I noticed that in org where the italics were barely visible in the theme I was trying (nord-theme from melpa).

**Comment by rougier** [comment]
I don't like italic too much because in a lot of font, it is simply replaced by oblique (https://creativepro.com/typetalk-italic-vs-oblique) and does not look right (in my eyes). I thus disable it.

**Comment by barrosfelipe** [comment]
I agree with @gutofarias that faded (low importance) and italics (emphasis) are opposites, so not really interchangeable semantically. I believe you have a valid point @rougier, in that the oblique fallback looks horrible anywhere, but many of the popular typefaces that are specialized either for body text or for code and freely available (and thus, widely used by Emacs users) implement proper italics, so disabling by default is not fair with the font designers that spent time implementing a widely accepted textual recourse. I work with typography a lot (web development) and even though I have no proof to back up this claim, typefaces that do not have italics are often shallow (only a small subset of characters, thus not good for Emacs) or not designed for body text (heavily stylized or made for designers and focused in few applications, like headers and titles). Nano is your baby and your work is mind blowing and rich, so I respect your decision to turn it off of course—I just find it a bit extreme.

**Comment by rougier** [comment]
I might reconsider this decision and I'm open to suggestion. I think we could re-establish italic in the italic face and that won't change things too much for nao-emacs (but I'm not too sure about this).

**Comment by SterlingHooten** [comment]
What I've appreciated about this design has been the empirical backing used by @rougier to guide decisions.
Specifically I found the argument that color is a strong UI channel which has been overcrowded due to syntax highlighting a convincing one. Building from this reasoning we could ask what other channels are available to us for conveying information (especially semantic rather than syntactic).

Oppenheimer in [Fortune Favors the Bold (and the Italicized)](https://escholarship.org/uc/item/4wd1s7hj) argues that the intentional _distortion present in italic fonts requires greater cognitive strain to decode_ and thus helps with retention of that data. Although I agree with @rougier in [On the Design of Text Editors](https://arxiv.org/abs/2008.06030) that italics are abused in quantity and purpose, it seems that lumping the italic channel in with Faded for secondary information reduces the expressiveness of the medium. Further, in the reading of text for which authors have correctly and consistently applied italics (e.g. definitions in mathematical texts) the intended effect is inverted.

In this particular case, with the Roboto Mono font, the italics appear closer to oblique in that I see little distinction other than slanting (please correct me). But perhaps this points to a mismatch in objectives between the font creator whose intent was to "optimized for readability on screens," and _my interpretation_ of this project as "an interface informed by empirical research rather than dogma."

Perhaps there could be a customize option in the package to classify italic text as Faded or italic.

**Comment by rougier** [comment]
I think you're right in your analysis of the faux-italic in Roboto that might have influenced my decision to get rid of italic. I recently discovered the Victor Mono font (https://rubjo.github.io/victor-mono/) with a lovely italic and I think I'm now convinced that italic might a a real added value (but not with Roboto). I would need to search for font with real italic and stat to experiment. Or do you have already some pointers by any chance ? And I agree with your proposal of having an option for italic such that user can choose. Do you think you could make a PR?

Thank you very much for the Oppenheimer article.
